templates/backup-pre-mysql.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+
+# Exits after first failure
+set -e
+
+TMPDIR="{{ MYSQL_SERVER_BACKUP_DIR }}"
+timestamp=$(date +%Y%m%dT%H:%M:%S)
+
+rm -rf "${TMPDIR}"
+mkdir -p "${TMPDIR}"
+mysql -s -e 'show databases' | tail -n +1 | grep -v performance_schema | grep -v information_schema | while read DBNAME; do mysqldump -uroot --complete-insert --events --single-transaction "${DBNAME}" > "${TMPDIR}/${DBNAME}".sql; done
```

Comment:
You are right, this is a noop here. Suprisingly: `mysql -s -e 'show databases' -u root -p` shows database header while `mysql -s -e 'show databases' -u root -p | tail -n +1` does not, which is probably due to the fact that mysql is being smart, and tries to detect interactive prompt. 

I'll use `--skip-column-names` switch instead, which will filter `'Databases'`column.